subprojects/language-java/src/testFixtures/groovy/org/gradle/language/fixtures/ServiceRegistryProcessorFixture.groovy

```diff
@@ -57,7 +57,7 @@ try {
         writer.close();
     }
 } catch (IOException e) {
-    messager.printMessage(Diagnostic.Kind.ERROR, "Failed to generate source file " + className);
+    messager.printMessage(Diagnostic.Kind.ERROR, "Failed to generate source file " + className + ". " + e.getMessage());
```

Comment:
Just checking if having this change in the final version was your intention.

Reply:
Yes, because I faced incomplete error messages during debugging sessions.